hub/api/dataset.py

```diff
@@ -189,8 +192,31 @@ def load(
     @staticmethod
     @hub_reporter.record_call
     def delete(path: str, force: bool = False, large_ok: bool = False) -> None:
```

Comment:
@mccrearyd does it make sense to add the large `large_ok` argument to `ds.delete()` too?

Reply:
i would prefer that ds.delete() only works for small, and the error message tells them to explicitly call the static method

Reply:
because ds.delete(large_ok=True) is way easier to copy paste than the static method where they type explicitly the path

Reply:
but this isn't a big deal